labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -64,9 +79,12 @@ object CollectionExercise03 {
    * checkValuesIncrease(Seq(1,2,3)) == true
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
-  def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean =
-    error("fix me")
-
+  def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean = {
+   var result = true;
+   if (seq.size > 1)
+     seq.sliding(2).foreach(pair => result = result && pair(0) < pair(1))
+   result
```

Comment:
Took me a moment to figure out what was happening. You did a good job implementing `fold` on the fly. Even more conveniently, this particular case for "All these Booleans are true" is provided in Scala by `forall`, mirroring Python's `all`. Python's `any` is, I believe, `exists`.